byron/ledger/executable-spec/test/Ledger/Upiec/Properties.hs

```diff
@@ -0,0 +1,128 @@
+{-# LANGUAGE TypeApplications #-}
+
+module Ledger.Upiec.Properties
+  ( noProtVerChange
+  , protVerChangeAdopt
+  , protVerChangeSameComponents
+  , protVerChangeEmptyComponents
+  ) where
+
+import           Control.Lens (_1, _2, _3, _4, _5, _6, _7, _8, _9, (^.))
+import           Control.State.Transition (applySTS, TRC(..))
+import           Data.Map.Strict (Map)
+import qualified Data.Map.Strict as Map
+import qualified Data.Set as Set
+import           Ledger.Core (Slot)
+import           Ledger.Update (UpId)
+import qualified Ledger.Update.Generators as G
```

Comment:
Maybe call this something like `UpdateGen` instead of `G`, otherwise we have to scroll up to see what `G` means.

Reply:
`G` is the only generator module imported in the `Ledger.Upiec.Properties` module, which is 129 lines long. Therefore, there is no need to be more specific here, is there?

Reply:
Normally when I read code I don't go thought the import list and load the qualified imports into my head. So when I read `G` I had to go back and look. If we would have written `UpdateGen` this wouldn't have been necessary.

Reply:
`UpdateGen` it is.